pom.xml

```diff
@@ -177,7 +177,6 @@
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpcore</artifactId>
       <version>4.4</version>
-      <scope>test</scope>
```

Comment:
At commit 0cb6fc2a8b818956f8a2b8d1de85abb4f477e186 (currently master), I get [this error](https://gist.github.com/mantognini/a42137016dc927c3da0f) when clicking on the annotate button of the demo.